contrib/IECoreAlembic/include/IECoreAlembic/IGeomParamTraits.h

```diff
@@ -90,6 +90,8 @@ IECOREALEMBIC_SPECIALISEGEOMPARAMTRAITS( Alembic::AbcGeom::IN2dGeomParam, IECore
 IECOREALEMBIC_SPECIALISEGEOMPARAMTRAITS( Alembic::AbcGeom::IN3fGeomParam, IECore::V3fVectorData, IECore::GeometricData::Normal )
 IECOREALEMBIC_SPECIALISEGEOMPARAMTRAITS( Alembic::AbcGeom::IN3dGeomParam, IECore::V3dVectorData, IECore::GeometricData::Normal )
 
+IECOREALEMBIC_SPECIALISEGEOMPARAMTRAITS( Alembic::AbcGeom::IBoolGeomParam, IECore::IntVectorData, IECore::GeometricData::Numeric )
```

Comment:
IECoreRI/PrimitiveVariableList.cpp doesn't like BoolVectorData:
https://github.com/ImageEngine/cortex/blob/6fa38cea55d5e8bde8dec85e8ae707a757712bec/src/IECoreRI/PrimitiveVariableList.cpp#L189

So for now, I've chose to just convert it to IntVectorData, to minimize changes.